### ISSUE TITLE
NTR: Eye: corneopalpebral synechiae (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7982]
New term request Eye: corneopalpebral synechiae (elthat_100906215938): Adhesion between eyelid and cornea

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000101
- label: Corneopalpebral synechiae
- parents: [Abnormal eyelid morphology (HP:0000492)](https://hpo.jax.org/app/browse/term/HP:0000492) | [Abnormal cornea morphology (HP:0000481)](https://hpo.jax.org/app/browse/term/HP:0000481)
- pmid: PMID:20301721
- def: An adhesion between the conjunctiva of the eyelid and the cornea.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7982
- synonyms:
